code/modules/atmospherics/machinery/valve.dm

```diff
@@ -206,6 +206,7 @@ obj/machinery/atmospherics/valve
 		else
 			src.open()
 			if(high_risk) message_admins("[key_name(user)] has opened the valve: [src] at [log_loc(src)]")
+		add_fingerprint(user)
```

Comment:
Consider use of `..()` instead, I think the use of `interact_particle()` helps the interaction stand out and should make teaching TEG other other valve based mechanics easier to follow instead of needing to point.

Reply:
On second thought maybe not, I don't think.  Maybe just add `interact_particle(user,src)` otherwise it opens the door to borg things and checking for brain damage.